aspnetcore/includes/managed-identities.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> Microsoft recommends that you use the most secure authentication flow available. If you're connecting to Azure SQL, [Managed Identities for Azure resources](/sql/connect/ado-net/sql/azure-active-directory-authentication#using-managed-identity-authentication) is the recommended authentication method.
```

Comment:
For asp.net core, this is far to weak. Something similar to my suggestion above.